cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java

```diff
@@ -1267,4 +1270,22 @@ public void testVisualSearch() {
         Util.processWriteParameters(options, params);
         assertEquals(true, params.get("visual_search"));
     }
+
+    @Test
+    public void testDeleteBackedupAsset() throws Exception {
```

Comment:
This feature requires some custom flag. Not sure it is enabled for all accounts. I would use the skip_test feature and make it conditional.